.pre-commit-config.yaml

```diff
@@ -15,10 +15,14 @@ repos:
     rev: 5.8.0
     hooks:
       - id: isort
-        args: ["--profile", "black"]
```

Comment:
Why are we removing them ?  Without using this argument, isort and black might conflict with each other. That's why it is important to use this in every project

Reply:
I added this argument in the tool.black section of the pyproject toml so that it simplifies pre-commit bc the two tools look at this file for their config anyways